spec/integration/session_spec.rb

```diff
@@ -905,6 +905,24 @@
         @session.visit '/poltergeist/with_js'
       end
 
+      it 'matches on partial strings' do
+        expect {
+          @session.accept_confirm 'match this' do
```

Comment:
Have the modal have some characters that have meaning in a Regexp so the escaping is correctly tested